lms/templates/admin/organizations.html.jinja2

```diff
@@ -14,5 +14,46 @@ Organizations
             New organization
         </a>
     </div>
```

Comment:
The "New organization" button shows up in the "Find organization" area, but doesn't use any of the fields. This should probably be outside of this box?